src/lib/datepicker/calendar-footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="mat-calendar-footer cdk-visually-hidden">
+  <div class="mat-calendar-controls">
+    <!-- The close button is for screen readers (it is invisible) -->
+    <button mat-button type="button" class="mat-calendar-close-button"
+            (click)="closeClicked()" [attr.aria-label]="closeCalendarLabel"
+            cdkAriaLive="polite">{{closeCalendarLabel}}
```

Comment:
The element doesn't need `cdkAriaLive` since screen readers will read it out when it receives focus.